doc/users/next_whats_new/selector_improvement.rst

```diff
@@ -0,0 +1,8 @@
+Selectors improvement
+---------------------
+
+The `~matplotlib.widgets.RectangleSelector` and
+`~matplotlib.widgets.EllipseSelector` can now be rotated interactively. The rotation is
+enabled and disabled by pressing the 'r' key.
+The "square" shape can be defined in data or display coordinate system as defined
+by the *data_coordinates* modifier.
```

Comment:
From a quick look at the code rotation and data_coordinate both seem to be modifiers. You should then use the same type of description for both - i.e. rather say "d key" instead of *data_coordinates* modifier. Or mention the modifier in both.